acct/check

```diff
@@ -132,11 +135,53 @@ if [ -f "$vhost_file" ] && vhost=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$v
 fi
 
 # check apphosting config
-if [ -f "$vhost_app_file" ] && vhost=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_app_file"); then
+if [ -f "$vhost_app_file" ] && vhost_app=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_app_file"); then
   echo
   echo "Virtual apphosts:"
   echo "-------------------------------------------------------------------------------"
-  echo "$vhost"
+  echo "$vhost_app"
+  echo "-------------------------------------------------------------------------------"
+fi
+
+# check virtual mail config
+if [ -f "$vhost_mail_file" ] && vhost_mail=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_mail_file"); then
+  echo
+  echo "Virtual mail:"
+  echo "-------------------------------------------------------------------------------"
+  echo "$vhost_mail"
+  echo "-------------------------------------------------------------------------------"
+fi
+
+# check dns entries
+expand_fqdns()
+{
+  while read -r entry; do
+    if [ -n "$entry" ]; then
+      username=$(echo "$entry" | cut -d' ' -f1)
+      domain=$(echo "$entry" | cut -d' ' -f2)
+
+      if [ "$domain" = "-" ]; then
+        echo "$username.berkeley.edu"
+      elif echo "$domain" | grep -qE '.+\.[a-z]+$'; then
+        echo "$domain"
+      else
+        echo "$domain.berkeley.edu"
+      fi
+    fi
+  done
```

Comment:
:(

could we put this new stuff in a python script and call it? it kinda sucks to be re-implementing the parsing logic

Reply:
That's true, it would be nice to have the same parsing code for this too...

Reply:
I switched this out for a python script